tensorboard/webapp/runs/views/runs_table/regex_edit_dialog.ng.html

```diff
@@ -14,23 +14,38 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<h1 mat-dialog-title>Color runs by regex</h1>
+
 <mat-dialog-content>
+  <p>Enter a regex with capturing groups to match against run names:</p>
   <mat-form-field>
     <input
       matInput
       #regexStringInput
-      placeholder="(\w+/).+"
       value="{{ regexString }}"
       (keydown.enter)="onEnter($event.target.value)"
       cdkFocusInitial
     />
   </mat-form-field>
 </mat-dialog-content>
 
-<div mat-dialog-actions>
+<div class="example-details">
+  <p>
+    Each matching run will be assigned a color based on the "key" formed by its
+    matches to the capturing groups. <br />
+    <button mat-stroked-button (click)="fillExample()">
```

Comment:
Perhaps fill the argument and write `"fillExample('(train|eval)')"` so the example regex all in HTML.

Reply:
We might want to consider just using a link (i.e. underline and maybe also blue coloring) for the live example?  The inline button is a little cramped-looked IMO, and I think a link would still be fairly obviously a click target.

Reply:
- updated with '(train|eval)' in HTML
- replaced button with plain text link